client/src/components/spinner.component.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { describe, expect, it } from 'vitest';
+
+import { AstrosatSpinner } from './spinner.component';
+
+import { render, screen } from '~/test/utils';
```

Comment:
This is not going to stop you from progressing, the code now looks in pretty good shape. I expected eslint/prettier to order the imports better though, I expected:

1. 3rd party imports from react,vite,react-router-dom etc
2. Absolutely referenced local code e.g. `~/blah`
3. Relatively refrenced local code e.g. `./blah`

The idea is for imports to be imported in a consistent manner, so it is easy to look at a section to see imports. I don't think this is a problem with what you've done though, just to be clear. Either you don't have a `.vscode/settings.json` file in your repo, in fact I've just checked and this is true, I did say this to everyone last week, or the week before, you obviously missed the message in the dev channel.

This issue shouldn't prevent you from merging this code, but you should copy the `.vscode` directory from another project, as I will just ignore PRs that are not following our Codign Standards, in other PRs. If it turns out our eslint/prettier settings are not capturing this requirement to order imports, then we have a bug that needs fixed urgently.